<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>englishResults</class>
 <widget class="QWidget" name="englishResults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #4486c3;</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:0px;
border-top-right-radius: 25px;
border-bottom-right-radius: 25px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="exams_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/exams_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="program_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/program_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="results_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/results_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="inbox_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/inbox_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profile_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/images/profile.icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logout_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>630</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/assets/images/log_out_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="examsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>195</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Exams</string>
    </property>
   </widget>
   <widget class="QPushButton" name="programButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sadas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inboxButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Inbox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="profileButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>580</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="navigationLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 17pt;
color:white;</string>
    </property>
    <property name="text">
     <string>NAVIGATION</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoIcon">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>-10</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/assets/images/Logo1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 600 12pt &quot;Crimson Text Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UNIVERSE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lessons_Icon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources.qrc">:/assets/images/lessons_icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lessonsButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>440</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Allerta&quot;;
color:white;
text-align:left;</string>
    </property>
    <property name="text">
     <string>Lessons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border-radius:35px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="mathsBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>801</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:0px;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="grammarTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 32pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;GRAMMAR&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="grammarBestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;BEST RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="grammarLatestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LATEST RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="grammarExam1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="grammarExam2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="grammarExam3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestGrammarExam1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestGrammarExam1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestGrammarExam2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestGrammarExam2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestGrammarExam3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestGrammarExam3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="mathsBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>410</y>
     <width>801</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:0px;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="vocabularyTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 32pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;VOCABULARY&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vocabularyBestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;BEST RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vocabularyLatestResult">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;LATEST RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vocabularyExam1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vocabularyExam2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="vocabularyExam3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;EXAM 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestVocabularyExam1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestVocabularyExam1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestVocabularyExam2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestVocabularyExam2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestVocabularyExam3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="latestVocabularyExam3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 20pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;RESULT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="resultsTitle">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(155, 192, 226);
border:0px;
border-radius: 20px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>4</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font: 37pt &quot;Aura&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ENGLISH RESULTS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
